code/authservice/src/main/java/com/swe6813/team4/authservice/SecurityConfig.java

```diff
@@ -2,13 +2,33 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.lang.NonNull;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 @Configuration
 public class SecurityConfig {
+  //  TODO: ADD PROD ORIGIN
+  public static String[] ALLOWED_ORIGINS = {"http://localhost:3000"};
```

Comment:
Curious, is there a way to read this from like a yaml file or something so the program doesn't have to be recompiled to add origins?

Reply:
moved to `application.properties`!